src/__configuration__/resources/index.tsx

```diff
@@ -54,6 +54,14 @@ export const resources: ResourceBucket[] = [
                 repository: 'icons',
                 bugLabels: ['font'],
             },
+            {
+                name: '@pxblue/react-native-vector-icons',
+                description: 'Vector icons for React Native',
+                package: '@pxblue/react-native-vector-icons',
+                repository: 'icons',
+                applies: ['react-native'],
+                bugLabels: ['react-native'],
+            },
```

Comment:
can we move this to the bottom of the list next to the rn progress icons?

Reply:
Just moved this down, should be good now.